src/Core/Resolvers/MsSqlQueryBuilder.cs

```diff
@@ -256,5 +256,16 @@ public string BuildStoredProcedureResultDetailsQuery(string databaseObjectName)
                             "WHERE is_hidden is not NULL AND is_hidden = 0";
             return query;
         }
+
+        /// <inheritdoc/>
```

Comment:
The implementation here is specific to SQL Server. Instead of `<inheritdoc/>`, describe what is considered a read-only column in sql server: `timestamp` / `is_computed`. And noting (If i recall from a previous screenshot you provided) the `timestamp` field has the trait `is_computed=0`.